### ISSUE TITLE
Install fails because of clang error

### ISSUE THREAD

**Issue by mhantke** [ghisvail/pyNFFT#51]
I tried to install with pip / easy_install on a Mac (OS X Yosemite 10.10.5) and the installation fails due to the following clang error:

clang: error: unknown argument: '-malign-double'

The compiling option "-malign-double" is not supported by clang and instead of a warning an error is raised.

**Comment by christianhaargaard** [comment]
I don't know if this is a correct way to deal with it, but I downloaded the library.
Edited `setup.py` and changed `-malign-double` to `-align`, and then ran `pip install -e .` from within the library to install. I haven't tested if the library works as intended as I just learned about it, but it does install.

**Comment by ghisvail** [comment]
Yes there is a high chance that the current content of my `setup.py` uses options which are not portable to `clang`. Feel free to apply @christianhaargaard suggestion, run the test suite and report whether it was successful.

**Comment by asarnow** [comment]
See my comment in #53, best thing to do is use Homebrew to install gcc-5 or clang-omp and use one of those compilers instead of Xcode's clang. Homebrew also makes it trivial to obtain pyNFFT's dependencies.
